### PR TITLE
Small fixes to spliterd man page and --peer option

### PR DESCRIPTION
**Pull request by agunde406**

### CODE REVIEW

File: splinterd/man/splinterd.1.md [hunk 1]
```diff
@@ -159,10 +159,11 @@ OPTIONS
 `--peer PEER-URL` `[,...]`
 : Specifies one or more Splinter nodes that `splinterd` will automatically
   connect to when it starts. The *PEER-URL* argument must specify another node's
-  network endpoint, using the format `ip:port`.
+  network endpoint, using the format `protocol_prefix://ip:port`.
 
   Specify multiple nodes in a comma-separated list or by repeating the
-  `--peer` option.
+  `--peer` option. The protocol prefix part of the peer URL will specifies the
```
Comment (ltseeley) [inline splinterd/man/splinterd.1.md:165]: Typo: specifies -> specify

Comment (agunde406) [reply splinterd/man/splinterd.1.md]: reworded

File: splinterd/man/splinterd.1.md [hunk 2]
```diff
@@ -134,6 +134,9 @@ OPTIONS
   Any options on the command line will override the settings in the
   configuration file.
 
+`--display-name DISPLAY-NAME`
+: Specifies a human-readable name for the node (Default: "")
```
Comment (ltseeley) [inline splinterd/man/splinterd.1.md:138]: The default value for display name was actually updated to `Node <node_id>`

Comment (agunde406) [reply splinterd/man/splinterd.1.md]: updated

**Pull request merged**